From: Martin Wilck <mwilck@suse.com>
Subject: scsi: smartpqi: set force_blk_mq=1

With the default single-queue block layer in SLE12, smartqpi will only
use a single queue/IRQ. Switch to blk-mq for this driver to avoid it.

Signed-off-by: Martin Wilck <mwilck@suse.com>
Patch-mainline: Never, fixed in newer kernels by defaulting to blk-mq
References: bsc#1179310
---
 drivers/scsi/smartpqi/smartpqi_init.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/scsi/smartpqi/smartpqi_init.c
+++ b/drivers/scsi/smartpqi/smartpqi_init.c
@@ -6571,6 +6571,7 @@ static struct scsi_host_template pqi_dri
 	.scan_finished = pqi_scan_finished,
 	.this_id = -1,
 	.use_clustering = ENABLE_CLUSTERING,
+	.force_blk_mq = 1,
 	.eh_device_reset_handler = pqi_eh_device_reset_handler,
 	.ioctl = pqi_ioctl,
 	.slave_alloc = pqi_slave_alloc,
